### PR TITLE
add command on thicket-webapp readme to make easily installation

### PR DESCRIPTION
**Pull request by josselinchevalay**
#307

### CODE REVIEW

File: packages/thicket-webapp/README.md [hunk 1]
```diff
@@ -10,7 +10,13 @@ By setting the corresponding `env` var the IPFS instance will connect to the def
 - `WRTC_IPFS` - will use IPFS’s [`webrtc-star-signal`](https://wrtc-star.discovery.libp2p.io) server. Default value if no value is set.
 
 ```sh
+# to be sure have nodejs 8
```
Comment (AquiGorka) [inline packages/thicket-webapp/README.md]: Would you please double check this?
I ran `nvm use 9` then `yarn && yarn bootstrap` and it worked correctly.
(it used node v9.11.2)

File: packages/thicket-webapp/README.md [hunk 2]
```diff
@@ -10,7 +10,13 @@ By setting the corresponding `env` var the IPFS instance will connect to the def
...
+#goto root project
```
Comment (AquiGorka) [inline packages/thicket-webapp/README.md]: Would you mind updating your PR to the following texts:

Requires node version __ (let's confirm the correct version)

```sh
# please make sure you have bootstrapped all the packages accordingly
# from repo root run (cd ../../)
yarn
yarn bootstrap

# then in packages/thicket-webapp (cd packages/thicket-webapp)
yarn start
```

File: packages/thicket-webapp/README.md (hunk 1 above)
Comment (AquiGorka) [reply packages/thicket-webapp/README.md]: thicket-docker

I double checked with a setting similar to yours (using docker image for node9)

File: packages/thicket-webapp/README.md (hunk 2 above)
Comment (josselinchevalay) [reply packages/thicket-webapp/README.md]: yep make sense. do you want update my change with that ?

**Pull request merged**